### DISCUSSION TITLE
What does "reversible" mean?

### DISCUSSION THREAD

**Discussion by AlexanderZeilmann** [geoopt/geoopt#159] [Q&A] [answered]
I am currently implementing a manifold and encountered the ```reversible``` flag. What does it mean? I have not found any documentation about it.

**Comment by ferrine** [comment, marked as answer]
It means reversible combination of retraction and vector transport.
p==exp(exp(p, v), -T_{p, exp(p, v)}(v))

**Comment by ferrine** [comment]
In case the combination is reversible, the manifold can be used in mcmc
